prisma/schema.prisma

```diff
@@ -10,8 +10,39 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-model Employee{
-  employeeId Int @id @default(autoincrement())
-  name String
-  salary Int @default(100000)
+model Employee {
+  employeeId   Int        @id @default(autoincrement())
+  name         String
+  salary       Int        @default(100000)
+  designation  Designation @default(Junior)
+  department   Department @relation(fields: [departmentId], references: [departmentId])
+  departmentId Int
+  projects ProjectAssignment[]
+}
+
+model Department {
+  departmentId    Int        @id @default(autoincrement())
+  departmentName String
+  Employee       Employee[]
+}
+
+model Project {
+  projectId Int @id @default(autoincrement())
+  projectName String
+  employees ProjectAssignment[]
```

Comment:
wrong field name

Reply:
My thinking was projects contains employees those are assigned with a project. Should it be "assignedEmployees"?

so,
 project -> employees

employees -> projects